cfd_protocol/src/protocol/transactions.rs

```diff
@@ -496,3 +494,42 @@ pub fn punish_transaction(
 
     Ok(punish_tx)
 }
+
+#[derive(Clone, Debug)]
+struct Fee {
+    fee: f64,
+}
+
+impl Fee {
+    pub fn new(signed_vbytes: f64) -> Self {
+        let fee = signed_vbytes * SATS_PER_VBYTE;
+        Self { fee }
+    }
+
+    /// Note: does not mutate, but creates another Fee object
+    pub fn add(&self, number: f64) -> Fee {
+        Fee {
+            fee: self.fee + number,
+        }
+    }
+
+    pub fn rounded(&self) -> u64 {
+        self.fee.ceil() as u64
```

Comment:
❓ This `ceil` is for the minimum relay fee, so with `ceil`ing we ensure that the minimum is always met, correct?
Just wanna make sure my understanding is correct :)

Reply:
yes, I'll add a comment reflecting that.